src/status_im2/contexts/onboarding/enable_biometrics/view.cljs

```diff
@@ -37,13 +41,34 @@
        :style                     {:margin-top 12}}
       (i18n/label :t/maybe-later)]]))
 
+(defn enable-biometrics-parallax
+  []
+  [:<>
+   [parallax/video
+    {:layers  (:biometrics resources/parallax-video)
+     :stretch 50}]
+   [rn/view
+    [navigation-bar/navigation-bar {:disable-back-button? true}]
+    [page-title]]])
+
+(defn enable-biometrics-simple
+  []
+  [:<>
+   [rn/view
+    [navigation-bar/navigation-bar {:disable-back-button? true}]
+    [page-title]]
+   [rn/view {:style style/page-illustration}
+    [rn/image {:source (resources/get-image :biometrics)}]]])
+
 (defn enable-biometrics
   []
   (let [insets (safe-area/get-insets)]
     [rn/view {:style (style/page-container insets)}
-     [navigation-bar/navigation-bar {:disable-back-button? true}]
-     [page-title]
-     [rn/view {:style style/page-illustration}
-      [quo/text
-       "Illustration here"]]
-     [enable-biometrics-buttons {:insets insets}]]))
+     [background/view true]
+     (if whitelist/whitelisted?
+       [enable-biometrics-parallax]
+       [enable-biometrics-simple])
+     [enable-biometrics-buttons {:style {:align-self :flex-end}}]]))
```

Comment:
We're passing a map of `{:style ...}` and receiving ```clj :insets``` key in `enable-biometrics-buttons` function.
Is that a typo?
And if we need these styles can't we have them inside the `enable-biometrics-buttons` function's main view?

Reply:
uff, yeah good spot! I will update this :)